### PR TITLE
Revert Changes that add attributes to scope.

### PR DESCRIPTION
**Pull request by MadVikingGod**
As was discussed in the spec meeting.  This change reverts the addition of attributes to scope, to not get ahead of the specification and unblock release of v1.10.

This will unblock #3145.  If these changes are added back, #3145 should block the v1.11 release.

Revert "Add instrumentation scope attributes (#3131)"

This reverts commit 0078faeb0e84d44dced8230b251b260fb2b912e5.

Revert "Add WithScopeAttributes MeterOption to metric API package (#3132)"

This reverts commit 81a9bab814231c386bfee8078a66f13bd457288d.

### CODE REVIEW

**Review by dashpole — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

**Review by Aneurysm9 — APPROVED** [review]

**Review by jmacd — APPROVED** [review]

**Review by dmathieu — APPROVED** [review]

**Comment by MadVikingGod** [comment]
I am merging this a bit early, to unblock the 1.10 release.  It has overwhelming approval, and was also discussed at the SIG.

**Pull request merged**
